_posts/2023-12-24-update.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Week Ending December 24, 2023
+date: 2023-01-26 22:00:00 -0000
+slug: 2023-01-24-update
+---
+
+## Developer News
+
+SIG Release [updated](https://groups.google.com/a/kubernetes.io/g/dev/c/JfZQhip-R7U) its roadmap for 2024 onwards. If you want to contribute, just reach out to SIG Release using the K8s Slack (#sig-release) or comment on any of the open GitHub issues.
+
+## Release Schedule
+
+**Next Deadline: 1.30 Cycle Begins, January 2024**
+
+We are in the holiday period between releases; dates for 1.30 have not been published yet. If you want to become a part of the [Release team](https://github.com/kubernetes/sig-release/issues/2377), apply soon! Shadow applications are open until Wednesday, January 3rd 2024 and will be reviewed until January 12th 2024. Use this [form](https://docs.google.com/forms/d/e/1FAIpQLScTWKYMqO8TUKjqqvQzmjZ1y03mefrKqrAiPvhVqkvdzVrHAQ/viewform) to apply. 
+
+## Featured PRs
+
```

Comment:
Could mention https://github.com/kubernetes/website/pull/44478 - I'd appreciate having that once announced.

Reply:
Yes, I was planning to add it today under featured PRs!